pandas/core/groupby/generic.py

```diff
@@ -908,11 +908,15 @@ class DataFrameGroupBy(GroupBy):
     )
     @Appender(_shared_docs["aggregate"])
     def aggregate(self, func=None, *args, **kwargs):
-
         relabeling = func is None and is_multi_agg_with_relabel(**kwargs)
         if relabeling:
+            # OWO CHANGES
+            import json
+            for k, v in list(kwargs.items()):
+                if (isinstance(v[0], list)):
```

Comment:
want to use is_list_like for consistency

Reply:
Implemented